payloads/external/Makefile.inc

```diff
@@ -207,6 +207,7 @@ $(obj)/UEFIPAYLOAD.fd edk2: $(DOTCONFIG) $(IPXE_EFI)
 		CONFIG_EDK2_DASHARO_POWER_CONFIG=$(CONFIG_EDK2_DASHARO_POWER_CONFIG) \
 		CONFIG_EDK2_DASHARO_POWER_CONFIG_FAN_CURVE=$(CONFIG_EDK2_DASHARO_POWER_CONFIG_FAN_CURVE) \
 		CONFIG_EDK2_DASHARO_NETWORK_BOOT_DEFAULT_ENABLE=$(CONFIG_EDK2_DASHARO_NETWORK_BOOT_DEFAULT_ENABLE) \
+		CONFIG_EDK2_DASHARO_USB_CONFIG=$(CONFIG_EDK2_DASHARO_USB_CONFIG) \
```

Comment:
This line is already present above: https://github.com/Dasharo/coreboot/pull/346/files#diff-e9897668de3b042db9e1790c4ab68359db4c1a7dc392142e8e8b3725b00c14c7R204

Reply:
This was pushed by accident and no longer present. Now this PR has correct changes. Closing.